### DISCUSSION TITLE
`saveMissing: true` seems not working on server-side

### DISCUSSION THREAD

**Discussion by jorisre** [sergiodxa/remix-i18next#99] [Q&A]
I followed the Readme all is working except for `saveMissing: true` on server-side only (it works for client-side).

When I visit the home page (`app/routes/index.tsx`) I'm getting the following logs : 

>i18next::backendConnector: No backend was added via i18next.use. Will not load resources.
>i18next: languageChanged fr
>i18next::backendConnector: No backend was added via i18next.use. Will not load resources.
>i18next::translator: missingKey fr home page-title Default Title

I think `No backend was added via i18next.use. Will not load resources.` it's caused because my RemixI18Next does not implement my backend _(in my case Locize)_ when I do `i18n.getFixedT(...)` in `app/routes/index.tsx`

Does someone know how to solve this problem? thanks !

My server-side setup: 

```tsx
// entry.server.tsx

export default async function handleRequest(
  request: Request,
  responseStatusCode: number,
  responseHeaders: Headers,
  context: EntryContext
) {
  let instance = createInstance();

  let lng = await i18n.getLocale(request);
  let ns = i18n.getRouteNamespaces(context);

  await instance
    .use(initReactI18next)
    .use(Locize)
    .use(locizeLastUsed)
    .init({
      ...config,
      lng,
      ns,
      backend: {
        projectId: process.env.LOCIZE_PROJECT_ID,
        apiKey: process.env.LOCIZE_API_KEY,
        referenceLng: 'fr',
        allowedAddOrUpdateHosts: ['localhost'],
        translatedPercentageThreshold: 0.8,
      },
      locizeLastUsed: { projectId: process.env.LOCIZE_PROJECT_ID },
    });

  let markup = renderToString(
    <I18nextProvider i18n={instance}>
      <RemixServer context={context} url={request.url} />
    </I18nextProvider>
  );

  responseHeaders.set('Content-Type', 'text/html');

  return new Response('<!DOCTYPE html>' + markup, {
    status: responseStatusCode,
    headers: responseHeaders,
  });
}
```

```ts
// i18n.server.ts
import { RemixI18Next } from 'remix-i18next';
import { config } from './config';

export let i18n = new RemixI18Next({
  detection: {
    supportedLanguages: config.supportedLngs,
    fallbackLanguage: config.fallbackLng,
  },
  // This is the configuration for i18next used
  // when translating messages server-side only
  i18next: {
    ...config,
  },
});
```

**Comment by sergiodxa** [comment]
Did you configured a backend when creating an instance of RemixI18Next?

**Reply by jorisre** [reply]
I've updated the discussion with my setup. the backend is configured in `entry.server.tsx`
